### ISSUE TITLE
Is it still in active development?

### ISSUE THREAD

**Issue by Mastartiq** [astrofox-io/astrofox#110]
Will Astrofox be developed further and come with new features?

It has all the potential to be the best free and open-source software for audio visualisers.

It would be nice to see some implementation of VFX automation, FX models integration in different formats (e.g Open FX), video formats support, GIF support, improved editing support in the scenes and controls modes, the possibility to add our own Geometry 3D models and as well to add our own fonts instead of choosing just the ones provided.

**Comment by bartoszkrawczyk2** [comment]
Looks like the project is dead...

**Comment by mikecao** [comment]
Yes — Astrofox 2.0 has just been released with a full rewrite (new compositor, offline ffmpeg export, MIDI, live mode, radial spectrum, i18n, and a plugin system for custom displays/effects). Closing this question; the individual feature ideas here are tracked in their own issues.
